src/LanguageServer.php

```diff
@@ -125,6 +125,12 @@ public function initialize(ClientCapabilities $capabilities, string $rootPath =
             $this->indexProject()->otherwise('\\LanguageServer\\crash');
         }
 
+        if (extension_loaded('xdebug')) {
+            setTimeout(function () {
```

Comment:
I'm not sure if its best way to do this. This is added because tests are failing (notification message is first before initialization result).